stl/src/xcharconv_tables_double.cpp

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+// This must be as small as possible, because its contents are
+// injected into the msvcprt.lib and msvcprtd.lib import libraries.
+// Do not include or define anything else here.
+// In particular, basic_string must not be included here.
+
+#include <xcharconv_tables.h>
+
+namespace std {
+
+    /* static */ const uint64_t _General_precision_tables_2<double>::_Special_X_table[195] = {0x3F18E757928E0C9Du,

```

Comment:
We conventionally avoid commented-out code (`/* static */`, two places). I'm fine with cleaning this up in a followup if we can avoid resetting testing.